pkg/config/validation/validation.go

```diff
@@ -3505,6 +3505,10 @@ func validateLocalityLbSetting(lb *networking.LocalityLoadBalancerSetting) error
 		return err
 	}
 
+	if (len(lb.GetFailover()) != 0 || len(lb.GetFailoverPriority()) != 0) && outlier == nil {
+		return fmt.Errorf("outlier detection poicy must be provided for failover")
```

Comment:
This is a breaking API change

Reply:
yeah, I can switch to warning level.  It's probably better that way anyways - I can also add the warning to the MeshConfig setting as well